vortex-duckdb-ext/src/scan.rs

```diff
@@ -74,43 +81,63 @@ impl TableFunction for VortexTableFunction {
         }
 
         Ok(VortexBindData {
+            first_file,
             _column_names: column_names,
             _column_types: column_types,
         })
     }
 
-    fn function(
-        _bind_data: &Self::BindData,
-        _local_init: &mut Self::InitLocalData,
-        global_init: &mut Self::InitGlobalData,
+    fn scan(
```

Comment:
maybe in the next PR, call this `next_chunk`? Or something, scan is not really correct here

Reply:
I'll meditate on that 🙂